src/Pact/Native.hs

```diff
@@ -459,7 +476,7 @@ c = mkTyVar "c" []
 map' :: NativeFun e
 map' i as@[app@TApp {},l] = gasUnreduced i as $ reduce l >>= \l' -> case l' of
            TList ls _ _ -> (\b' -> TList b' TyAny def) <$> forM ls (apply (_tApp app) . pure)
-           t -> evalError' i $ "map: expecting list: " ++ abbrev t
+           t -> evalError' i $ "map: expecting list: " <> text (abbrev t)
```

Comment:
Would like to address/delete `abbrev` in this PR -- It really isn't used very much, 7 times in this PR AFAICT. The idea was to prevent large terms from blowing up the logs -- is there some facility in ansi-wl-pprint to elide after some length? That would pretty much cover it and we could ditch `abbrev`.